extensions/wikia/LinkSuggest/LinkSuggest.php

```diff
@@ -66,8 +66,9 @@ function getLinkSuggest() {
 
 	$out = LinkSuggest::getLinkSuggest($wgRequest);
 
-	$ar = new AjaxResponse($out);
+	$ar = new AjaxResponse( $out );
 	$ar->setCacheDuration(60 * 60);
+	$ar->setVary( implode( ",",[ "Accept-Encoding", WebRequest::MW_WIKI_ID_HEADER ] ) );
```

Comment:
We're already Varying on `X-Mw-Wiki-Id` [here](https://github.com/Wikia/app/pull/16877/files#diff-5409d922c826c27900d04133bd41e52bR92), but AjaxResponse's use their own Vary logic so we need to add it here as well.